scripts/apps/authoring/authoring/index.ts

```diff
@@ -101,7 +101,10 @@ angular.module('superdesk.apps.authoring', [
     .directive('html5vfix', directive.Html5vfix)
     .directive('sdDashboardCard', directive.DashboardCard)
     .component('sdCharacterCount', reactToAngular1(CharacterCount, ['item', 'html', 'limit'], [], 'display: inline'))
-    .component('sdAuthoringIntegrationWrapper', reactToAngular1(AuthoringAngularIntegration, ['itemId', 'action'], []))
+    .component('sdAuthoringIntegrationWrapper', reactToAngular1(
+        AuthoringAngularIntegration, ['itemId', 'action', 'hideMonitoring', 'isExpanded'],
```

Comment:
Update terminology accross files so it's the following:
`fullWidth` / `setFullWidth` - for both authoring components
`monitoringHidden` / `setMonitoringHidden` - for monitoring

The reason for different names is to reduce coupling and limit information authoring receives about implementation details of the rest of the system.